iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/api/bicep/HasKeyword.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.arm.tree.api.bicep;
+
+import org.sonar.iac.arm.tree.api.ArmTree;
+
+public interface HasKeyword extends ArmTree {
```

Comment:
```suggestion
public interface HasKeyword {
```
Does not need to extend ArmTree

Reply:
Unfortunately yes, because of the register method.

Reply:
I could let it only extends `Tree`, but I'm not sure if that isn't more confusing.

Reply:
Than it's fine to keep it